NuRadioReco/framework/sim_station.py

```diff
@@ -129,3 +160,24 @@ def deserialize(self, data_pkl):
                 channel = NuRadioReco.framework.sim_channel.SimChannel(0, 0, 0)
                 channel.deserialize(channel_pkl)
                 self.add_channel(channel)
+
+    def __add__(self, x):
```

Comment:
typically one passes the argument `other` to an operator. Makes it easier to read the code